pytest_celery/message_brokers/redis_broker.py

```diff
@@ -1,23 +1,27 @@
+from __future__ import annotations
+
 from functools import cached_property
-from typing import List
 
 from kombu import Queue
+from redis.client import Redis
 from testcontainers.redis import RedisContainer
 
 from pytest_celery.message_brokers.message_broker import MessageBroker
+from pytest_celery.utils.compat import List
 
 
 class RedisBroker(MessageBroker):
     @property
     def queues(self) -> List[Queue]:
         pass
 
-    def __init__(self):
-        super().__init__(RedisContainer())
+    def __init__(self, container=None):
+        container = container or RedisContainer()
+        super().__init__(container)
 
     @cached_property
-    def client(self):
+    def client(self) -> Redis:
         return self.container.get_client()
 
-    def ping(self):
+    def ping(self) -> None:
         self.client.ping()
```

Comment:
Should this return None? Raise an exception when ping fails? What's the API contract here?